modules/post/windows/gather/enum_files.rb

```diff
@@ -75,44 +74,47 @@ def download_files(location, file_type)
       filename = "#{file['path']}\\#{file['name']}"
       data = read_file(filename)
       print_status("Downloading #{file['path']}\\#{file['name']}")
-      p = store_loot("host.files", 'application/octet-stream', session, data, file['name'], filename)
+      p = store_loot("host.files", 'application/octet-stream', session, data, file['name'], filename, nil, true)
```

Comment:
It would be better that give an option like `KEEP_ORIGIN_PATH` in the module so that the users have their choice to decide to keep the origin path info or not.